eventbus/src/main/java/pl/edu/uj/kimage/eventbus/CommandHandler.java

```diff
@@ -0,0 +1,4 @@
+package pl.edu.uj.kimage.eventbus;
+
+public interface CommandHandler extends EventListener {
+}
```

Comment:
empty block should be documented. Could you explain to me why logically CommandHandler extends EventListener?